src/lib/form.ts

```diff
@@ -64,5 +64,14 @@ export function form(...fields: (Writable<Field<any>> | Readable<Field<any>>)[])
 		return fields.find((f) => get(f).name === name);
 	}
 
-	return { subscribe, reset, validate, getField };
+	function summary(): Record<string, any> {
+		return fields.reduce((carry, field) => {
+			const fieldContent = get(field);
+			carry[fieldContent.name] = fieldContent.value;
```

Comment:
Should we return all the fields or only the valid fields?

Reply:
I thought of this as well!
I was thinking that it would probably match the expectations closest if we'd return all fields, no matter if valid or not, since the output represents the current state of the form.

We could throw if any field is invalid?
But that would be a bit too opinated for my taste since a user can simply check for `$myForm.valid` if validity is required, before calling `summary()`, still leaving the option of just retrieving the summary as is.

Reply:
I agree with you.

I just wanted to open the debate here in case someone had a good use case for that.